examples/2d/elixir_euler_vortex_mortar_split_shockcapturing.jl

```diff
@@ -0,0 +1,71 @@
+
+using OrdinaryDiffEq
```

Comment:
I think this file can just be renamed to `elixir_euler_vortex_mortar_shockcapturing.jl`